PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/pushimage/ui/SettingPanel.java

```diff
@@ -188,12 +215,18 @@ public void reset(PushImageRunConfiguration conf) {
         if (!MavenRunTaskUtil.isMavenProject(project)) {
             List<Artifact> artifacts = MavenRunTaskUtil.collectProjectArtifact(project);
             setupArtifactCombo(artifacts, conf.getTargetPath());
+        } else {
+            List<MavenProject> mavenProjects = MavenProjectsManager.getInstance(project).getProjects();
+            setupMavenProjectCombo(mavenProjects, conf.getTargetPath());
         }
 
         PrivateRegistryImageSetting acrInfo = conf.getPrivateRegistryImageSetting();
         containerSettingPanel.setTxtFields(acrInfo);
         if (Utils.isEmptyString(conf.getDockerFilePath())) {
-            containerSettingPanel.setDockerPath(DockerUtil.getDefaultDockerFilePathIfExist(project));
+            MavenProject prj = (MavenProject) cbMavenProject.getSelectedItem();
+            if (prj != null) {
+                containerSettingPanel.setDockerPath(DockerUtil.getDefaultDockerFilePathIfExist(prj.getDirectory()));
```

Comment:
just curious whether this part of code is already triggered by the select listener